MergePDF/PdfSelectorControl.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Drawing;
+using System.Windows.Forms;
+
+namespace MergePDF
+{
+    public partial class PdfSelectorControl : UserControl
+    {
+        private OpenFileDialog openFileDialog;
+        public string FileName { get; private set; }
+        public int Index { get; }
+
+        TextBox textBox;
+
+        public PdfSelectorControl(int index)
+        {
+            InitializeComponent();
+
+            openFileDialog = new OpenFileDialog
+            {
+                InitialDirectory = "c:\\",
+                Title = "Open PDF file",
+                Filter = "PDF files (*.pdf)|*.pdf",
+                FilterIndex = 2,
+                RestoreDirectory = true
+            };
+
+            Index = index;
+
+            CreateTextbox();
+
+            Controls.Add(textBox);
+            Controls.Add(CreateOpenButton());
+        }
+
+        private void CreateTextbox()
+        {
+            textBox = new TextBox
+            {
+                Location = new Point(126, 0),
+                Size = new Size(400, 20),
+                Visible = true,
+                ReadOnly = true
+            };
+        }
+
+        private Button CreateOpenButton()
+        {
+            Button button = new Button
+            {
+                Location = new Point(10, 0),
+                Text = $"Browse {Index} file",
```

Comment:
I would say: "Browse file `{Index}`", resulting in "Browse file 1" & "Browse file 2".